sbhs/static/css/base.css

```diff
@@ -57,9 +63,51 @@ html, body{
         column-width: 100%;
         }
 }
+
 @media (max-width: 400px) {
         .btn.filter {
         padding-left: 1.1rem;
         padding-right: 1.1rem;
         }
-}
\ No newline at end of file
+}
+
+/* messages */
```

Comment:
Should not this be in a separate css instead of base css?
But not an issue for now!